Makefile

```diff
@@ -42,4 +42,7 @@ rpm-build/.exists:
 	mkdir -p rpm-build
 	touch rpm-build/.exists
 
-
+# For devel convinience
+install:
+	sudo rm -rf dist/ build/ 
```

Comment:
To be more DRY, you could reuse the `clean` command (also add one more folder to its list to delete) and then do the install after that. I could see arguments both for and against. That's just something for you to consider, 👍  to this PR either way.